recipes/folly/all/conanfile.py

```diff
@@ -111,7 +107,7 @@ def _configure_cmake(self):
         if not self._cmake:
             self._cmake = CMake(self)
             self._cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
-            self._cmake.definitions["CXX_STD"] = self.settings.compiler.get_safe("cppstd") or "c++14"
+            self._cmake.definitions["CXX_STANDARD"] = self.settings.compiler.get_safe("cppstd") or "c++14"
```

Comment:
`CXX_STANDARD` - is a property of the target. https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html
So setting it here does nothing.

`CMAKE_CXX_STANDARD` - is the efault value for CXX_STANDARD property of targets.
https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html

But setting `CMAKE_CXX_STANDARD` in the folly library also doesn't work.
https://github.com/facebook/folly/blob/master/CMake/FollyCompilerUnix.cmake#L15-L17

And setting `CXX_STD` was the correct option here, because that's what folly uses.

Reply:
Seems like `self.settings.compiler.get_safe("cppstd")` always returned `None` on the CI and thus `c++14` was always used. However, the format of the `cppstd` setting is "14"/"17"/"20", NOT "c++14"/"c++17"/"c++20" as expected by folly. Will add the conversion.